### ISSUE TITLE
[Question] Git flow and work organization

### ISSUE THREAD

**Issue by malparty** [itssomething/scraping#8]
# Issue

The requirement included `Make regular commits and merge code using pull requests`. No pull request was found in the GitHub repository. 

Could you share how you organized, prioritized, and split the work?

**Comment by itssomething** [comment]
1. I may have not read the second half of that sentence, which mean I only "Make regular commits". I believe this won't be much difference since I only work at one feature at a time, I do not work with them in parallel
2.  In terms of priotization, I will try to prioritize the most important part first, or, the part that will make this the Minimum Viable Product. Take this application for example, the bare minimum will be the data scraping on a web page, so I prioritize like this:
- The most important part for the MVP will be the data scraping feature. It has to work with one page at least. So I will work on the single page scraping first
- Then I will make more updates to make this work with many pages. Starting with sequentially scraping, because it will somewhat satisfy the requirement, just not optimal
- Then I will make updates to optimize the scraping process. In this case, I split it into many background jobs for parallel processing
- After the aboves is completed, the application can scrape data correctly and optimaly. I will now proceed to work on the "other parts in the requirement but are less important ", for example: sign in, sign up, UI, styling, some extra fields for accessibility,...

**Comment by malparty** [comment]
Thanks for your insights 🙏
